src/pages/ProductDesc/ProductDesc.js

```diff
@@ -1,8 +1,115 @@
 import React from "react";
 import "./ProductDesc.scss";
+import { useState, useEffect } from "react";
+import RecommendProducts from "./components/RecommendProducts/RecommendProducts";
+import NewProducts from "./components/NewProducts/NewProducts";
 
 const ProductDesc = () => {
-  return <div>ProductDesc입니다.</div>;
+  useEffect(() => {
+    fetch("/datas/RECOMMENDPRODUCTS_DATA.json", {
+      method: "GET",
+    })
+      .then(res => res.json())
+      .then(RECOMMENDPRODUCTS_DATA => {
+        setRecommendProductsData(RECOMMENDPRODUCTS_DATA);
```

Comment:
then 메서드의 인자는 콜백함수이기 때문에, 매개변수는 간단히 data 정도로만 쓰셔도 됩니다.
```suggestion
      .then(data => {
        setRecommendProductsData(data);
```